src/components/Nav.jsx

```diff
@@ -13,63 +13,63 @@ const StyledNav = styled.nav`
   display: flex;
   align-items: center;
   justify-content: space-between;
-  background: white;
-`
+  background: var(--nav-background-color);
 
-const StyledMenuIcon = styled.button`
-  display: none;
-  @media (max-width: 992px) {
-    display: inline-block;
+  .hamburgerMenu {
+    display: none;
   }
-`
 
-const StyledInternalLinks = styled.ul`
-  display: flex;
-  align-items: center;
-  padding: 0;
-  list-style-type: none;
-  gap: 2rem;
-`
+  .internalLinks {
+    display: flex;
+    align-items: center;
+    padding: 0;
+    list-style-type: none;
+    gap: 2rem;
 
-const StyledBrigadeLogo = styled.img`
-  display: block;
-  height: auto;
-  min-width: 120px;
-
-  @media (max-width: 576px) {
-    min-width: 85px;
+    a {
+      color: var(--nav-internal-links-color);
+      text-decoration: none;
+    }
   }
-`
 
-const StyledDonateAndSocialsContainer = styled.div`
-  display: flex;
-  align-items: center;
-  list-style-type: none;
-  padding: 0;
-`
-const StyledExternalLinks = styled.ul`
-  display: flex;
-  align-items: center;
-  list-style-type: none;
-  gap: 2rem;
-  margin-right: 1.5rem;
-  padding: 0;
-`
+  .brigadeLogo {
+    display: block;
+    height: auto;
+    min-width: 7.5rem;
 
-const StyledSocialImg = styled.img`
-  min-width: 24px;
+    @media (max-width: 576px) {
+      min-width: 5.313rem;
+    }
+  }
+  .donateAndSocialsContainer {
+    display: flex;
+    align-items: center;
+    list-style-type: none;
+    padding: 0;
+  }
+  .externalLinks {
+    display: flex;
+    align-items: center;
+    list-style-type: none;
+    gap: 2rem;
+    margin-right: 1.5rem;
+    padding: 0;
+  }
+  .socialImg {
+    min-width: 1.5rem;
+  }
 `
 
 const Nav = () => {
   return (
     <StyledNav className="navbar" aria-label="main">
-      <StyledMenuIcon aria-label="open" className="menuIcon">
+      <button aria-label="open" className="hamburgerMenu">
         <img src={hamburgerOpenMenu} alt="open menu" aria-hidden="true" />
-      </StyledMenuIcon>
-      <StyledInternalLinks className="internalLinks">
+      </button>
+      <ul className="internalLinks">
         <li id="logoContainer">
           <Link to="/">
-            <StyledBrigadeLogo
+            <img
```

Comment:
Let's use [Gatsby Image](https://www.gatsbyjs.com/plugins/gatsby-plugin-image/) on this.